checks/noexec

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
I think >&2 is a bashism, but sh is dash on Debian ;)

Reply:
I think this works on Debian 12 at least:
```console
$ drr debian:12 /bin/sh
# echo test >&2
test
# echo test >&2 > /dev/null
# 
```